pipes/WDL/workflows/amplicon16S_analysis.wdl

```diff
@@ -1,28 +1,18 @@
+version 1.0
 ##Test 
-##09.19.22
+##11.16.22
+import "../tasks/tasks_16S_amplicon.wdl" as qiime 
```

Comment:
If you are `importing` a WDL task file `as qiime` then maybe that file should be renamed to `tasks_qiime.wdl`?

Reply:
I think I should keep the 16S_amplicon so i changed qiime --> 16S_amplicon. Let me know if you agree with this decision, if not maybe then I can change the tasks & workflow . wdl --> tasks_qiime && qiime_worfklow.wdl?